cfpb/sbl-frontend
# [Error pages] Use DSR Hero instead of CF.gov's CSS/HTML

Closes #203 

## Changes

- Implement 404/500 pages with the DSR Hero component
- Update repo dependencies to pull in recent updates to DSR
- ~~Scope creep: Update page header to show `LOGIN` instead of `User profile    LOGOUT` when user is not logged in~~

## How to test this PR

1. Visit http://localhost:8899/400
2. Verify things look good
3. Open browser Inspector -> Network -> Network request blocking
4. Add rule to block `*/v1/institutions/*`
5. Visit http://localhost:8899/user-profile
6. You should be redirected to http://localhost:8899/500
7. Verify things look good

## Screenshots
### Error pages
||Error 404|Error 500|
|---|---|---|
|Desktop|![full-404](https://github.com/cfpb/sbl-frontend/assets/2592907/40bbc1fa-0f2b-46b7-92f7-0f8d129472b0)|![full-500](https://github.com/cfpb/sbl-frontend/assets/2592907/8808a4a7-68f8-49d2-9bf3-2a6eb00320d0)|
|Mobile|![mobile-404](https://github.com/cfpb/sbl-frontend/assets/2592907/a67ab55f-d266-4c1c-a69a-f5e3c1e3f896)|![mobile-500](https://github.com/cfpb/sbl-frontend/assets/2592907/58bbaf7b-c0d5-45e8-853c-7abc6298a5ed)|
